src/dbcsr.h

```diff
@@ -0,0 +1,54 @@
+#ifndef DBCSR
+#define DBCSR
+
+#include "mpi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+    void c_dbcsr_init_lib();
+    
+    void c_dbcsr_finalize_lib_aux(MPI_Fint* fcomm);
+    
+    void c_dbcsr_finalize_lib(MPI_Comm comm)
+    {
+        MPI_Fint fcomm = MPI_Comm_c2f(comm);
+        c_dbcsr_finalize_lib_aux(&fcomm);
+    }
+  
+    void c_dbcsr_distribution_new(void** dist, MPI_Fint* fcomm, int* row_dist, int row_dist_size,
+                                  int* col_dist, int col_dist_size);
+
+    void c_dbcsr_distribution_release(void** dist);
+
+    void test(char* str);
```

Comment:
what is that `test(...)` function?

Reply:
I will delete this. It was debug function.